src/stcal/jump/twopoint_difference.py

```diff
@@ -291,7 +304,7 @@ def calc_med_first_diffs(first_diffs):
     row2, col2 = np.where(num_usable_groups == 2)  # locations of >= 4 usable group pixels
     if len(row2) > 0:
         two_slice = first_diffs[:, row2, col2]
-        median_diffs[row2, col2] = np.nanmin(two_slice)  # add median to return arr for these pix
+        median_diffs[row2, col2] = np.nanmin(np.abs(two_slice), axis=0)  # add med. to return arr
```

Comment:
Why do we want to use `np.abs()` here? This function is being used to find the median of the first differences, in order to guesstimate an approximate real countrate for a given pixel. If we eliminate negatives, we're going to bias the result to always be positive. For example, in a pixel that has no real source signal in it, the first differences will bounce around an average of zero (since zero is the real countrate), with an amplitude equal to the read noise, e.g. 10, -7, -2, 5, -1, 8, etc. We want the median to come out as close as possible to the real countrate, which in this case is zero, but if we take the abs value of all the entries, it'll come out significantly non-zero.

Reply:
For the median, when there are only two diffs left, the one with the smallest magnitude (i.e the smallest jump in any direction) should be chosen. So if they were [-10, 2], I would select 2 because it represents a smaller jump. You're right thought that this isn't doing the correct thing - should use np.abs() to find the location of the minimum, but then return the value with the sign attached.

Reply:
Ah, right. I was forgetting about the fact that this is only for the special case of having 2 diffs left to compare. In that case it makes sense, in terms of returning the one that has the smallest deviation.